scripts/squidex-minimal-azure-arm.json

```diff
@@ -0,0 +1,277 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "containerName": {
+      "defaultValue": "squidex",
+      "type": "String",
+      "metadata": {
+        "description": "The name of the Azure Container Instance Group"
+      }
+    },
+    "squidexDomain": {
+      "defaultValue": "",
+      "type": "String",
+      "metadata": {
+        "description": "The Domain/Sub-domain to be used in Squidex."
```

Comment:
Can you change this "Optional domain name (FQDN) if you want to host Squidex under a custom domain. If not defined, a domain name will be assigned."

Reply:
Done